map/src/map/EditableMarker.js

```diff
@@ -63,12 +64,35 @@ export default class EditableMarker {
                 }
             }
         });
+
+        const markerEventHandler = (e) => {
+            if (
+                e.target?.draggable &&
+                e.target?.alt === 'Marker' &&
+                e.target?.classList?.contains('leaflet-marker-draggable')
+            ) {
+                if (e.type === 'touchstart') {
+                    this.ctx.setPointContextMenu({});
+                    e.preventDefault();
+                }
+                if (e.type === 'touchend') {
+                    e.preventDefault();
+                    if (e.target?.title !== 'poly') {
+                        e.target?.dispatchEvent(new Event('contextmenu', e));
+                    }
+                }
+            }
+        };
+        document.addEventListener('touchstart', markerEventHandler, { passive: false });
+        document.addEventListener('touchend', markerEventHandler, { passive: false });
```

Comment:
We need only 1 eventHandler added to document.

Please move addEventListener to one-time useEffect(..., []) into component which permanently exists as only one instance. For example, into LocalClientTrackLayer.